docs/extensions-plugins/neovim/commands.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Pieces Neovim Plugin Commands
+description: Learn how to use the Pieces Neovim Plugin commands to streamline your workflow and boost productivity.
+---
+
+# Commands
+
+The Pieces Neovim plugin provides several commands to interact with Pieces OS. Here's a list of available commands:
+
+## General Commands
+
+### `:PiecesHealth`
+
+Check the health of Pieces OS.
+
+```vim
+:PiecesHealth
+```
+
+### `:PiecesOSVersion`
+
+Display the version of Pieces OS.
+
+```vim
+:PiecesOSVersion
+```
+### `:PiecesPluginVersion`
+
+Display the current version of the Pieces Neovim plugin.
+
+```vim
+:PiecesPluginVersion
+```
+
+### `:PiecesCopilot`
+
+Open the Pieces Copilot window in a split view. 
+
+```vim
+:PiecesCopilot
+```
+
+#### Using the Copilot Chat
+
+1. Press `i` to enter insert mode and type your message.
+2. Press `<Esc>` to exit insert mode.
+3. Press `<Enter>` to send the message to Pieces Copilot and see the response.
+
+### Copilot Commands
```

Comment:
This should be an h2 right? and then id double check what the `/change_model` should be then

Reply:
Yes I have updated this @mason-at-pieces